staging/src/k8s.io/code-generator/generate-internal-groups.sh

```diff
@@ -85,11 +86,11 @@ if [ "${GENS}" = "all" ] || grep -qw "conversion" <<<"${GENS}"; then
 fi
 
 if [ "${GENS}" = "all" ] || grep -qw "client" <<<"${GENS}"; then
-  echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${OUTPUT_PKG}/clientset"
+  echo "Generating clientset for ${GROUPS_WITH_VERSIONS} at ${CLIENTSET_OUTPUT_PATH:-${OUTPUT_PKG}/clientset}"
   if [ -n "${INT_APIS_PKG}" ]; then
-    ${GOPATH}/bin/client-gen --clientset-name ${CLIENTSET_NAME_INTERNAL:-internalversion} --input-base "" --input $(codegen::join , $(printf '%s/ ' "${INT_FQ_APIS[@]}")) --output-package ${OUTPUT_PKG}/clientset "$@"
+    ${GOPATH}/bin/client-gen --clientset-name ${CLIENTSET_NAME_INTERNAL:-internalversion} --input-base "" --input $(codegen::join , $(printf '%s/ ' "${INT_FQ_APIS[@]}")) --output-package ${CLIENTSET_OUTPUT_PATH:-${OUTPUT_PKG}/clientset} "$@"
```

Comment:
I'm not sure whether we should allow for changing `clientset` directory ? Perhaps it's better if everyone generates into the same directory ? Would be nice to know what @sttts thinks about that ?

Maybe we should leave it as it was and update import paths instead :)

Reply:
a fan of this idea. @sttts wdyt?

Reply:
We have old code, e.g. in apiextensions-apiserver where people depend on the location of the clientset package. That's why this customization was added.